pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -399,6 +399,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("get", "list", "update").Groups(kapiGroup).Resources("replicationcontrollers").RuleOrDie(),
 				authorizationapi.NewRule("get", "list", "watch", "create").Groups(kapiGroup).Resources("pods").RuleOrDie(),
 				authorizationapi.NewRule("get").Groups(kapiGroup).Resources("pods/log").RuleOrDie(),
+				authorizationapi.NewRule("create").Groups(kapiGroup).Resources("events").RuleOrDie(),
```

Comment:
@liggitt policy PTAL (the client running inside the deployer pod needs ability to create events to report hooks/etc.).